vmdb/lib/vmdb/util.rb

```diff
@@ -1,16 +1,14 @@
 module VMDB
   module Util
     # load all subclasses recursively
-    def self.eager_load_subclasses(klass)
+    def self.eager_load_subclasses(klass, pattern = nil)
       ActiveSupport::Dependencies.autoload_paths.each do |root|
-        Dir.glob(File.join(root, "#{klass.underscore}_*.rb")).sort.each do |file|
-          name = File.basename(file, '.*')
-          if name =~ /^#{klass.underscore}_[^_]*$/  # filter out sub-subclasses
-            require_dependency file
-          end
+        Dir.glob(File.join(root, (pattern || "#{klass.underscore}_*.rb"))).each do |file|
```

Comment:
can you move the pattern defaulting to its own line?  it's so cluttered here making it hard to read.